XenModel/Actions/Updates/DownloadUpdatesXmlAction.cs

```diff
@@ -48,8 +48,11 @@ public class DownloadUpdatesXmlAction : AsyncAction
         private const string PatchesNode = "patches";
         private const string ConflictingPatchesNode = "conflictingpatches";
         private const string RequiredPatchesNode = "requiredpatches";
+        private const string ContainedPatchesNode = "containedpatches";

```

Comment:
Comment: this will be removed later, because according to the current plans we won't need these